### DISCUSSION TITLE
Is there future plan for allow the sidebar docking top/bottom and tab-window

### DISCUSSION THREAD

**Discussion by NAXSISPRO61** [VSCodium/vscodium#874] [Ideas]
Hi, Folks , this is nax

i wonder , in vscode the top/bottom docking for sidebar is not in their future plan and i use vscodium ( 4 a reason u know ). 

Is there someone added this feature for the vscodium, All my 8 monitors are vertical and i always have to use vscodium sidebar closed .

**Comment by daiyam** [comment]
I think of VSCodium to be just a rebuild of VSCode without telemetry. All patches are done to make it useable.
You could put your own patches in `patches/user` and build your own VSCodium.
Personally, I'm using [my fork](https://github.com/zokugun/MrCode) of VSCodium with additional patches.
